source/administration/notices.rst

```diff
@@ -15,13 +15,28 @@ Administrator notices are used to inform System Admins when a new server version
 
 Administrator notices can be disabled in the **System Console > Notices** page.
 
+Admin Advisor notices
+^^^^^^^^^^^^^^^^^^^^^
+
+Admin Advisor notifications are included in Mattermost from v5.26. These notifications are designed to provide guidance and assist in choosing features to support your deployment as it scales.
+
+When these notifications are triggered, a System Admin receives a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The triggers include the number of registered users in your deployment reaching a predefined threshold, the number of posts made, and the number of channels you've created. For more information, you can view the source code.
+
+These notifications also give System Admins quick access to optionally contact support for additional assistance with a single click as well as links to useful resources for user management and provisioning to support large systems.
+
+Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, information such as the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are sent to us. This information is used to contact you as requested and to help us better understand your needs.
```

Comment:
```suggestion
Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, contact information such as the email address and name associated with your Mattermost account is sent to us, so we can contact you as requested and to help us better understand your needs. The number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are also sent to us to better understand your environment and hence provide more tailored support.
```

Reply:
^the wording isn't the best, but the intention with the changes is to make it clear right away that the information sent to us is used to allow us contact them, and the additional information allows us to offer more tailored/custom support.